manifests/examples/infra/additionalresources.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: cluster.kurator.dev/v1alpha1
+kind: Cluster
+metadata:
+  name: additionalresources
+  namespace: default
+spec:
+  infraType: aws
+  credential:
+    secretRef: quickstart
+  version: v1.23.0
+  region: us-east-1
+  network:
+    vpc:
+      cidrBlock: 10.10.0.0/16
+    podCIDRs:
+      - 192.168.0.0/16
+    serviceCIDRs:
+      - 10.96.0.0/12
+    cni:
+      type: calico
+  master:
+    replicas: 1
+    instanceType: t3.large
+  workers:
+    - replicas: 1
+      instanceType: t3.large
+  additionalResources:
+    - kind: ConfigMap
+      name: test-cm
+---
+apiVersion: v1
+data:
+  plugin.yaml: |-
+    apiVersion: v1
+    data:
+      test-key: test-value    
+    kind: ConfigMap
+    metadata:
+      name: test-cm
+      namespace: default
+kind: ConfigMap
```

Comment:
IIUC clusterresourceset only supports configmap and secret, then how secret should be set?

Do we have an explicit document saying only Secret and CM are allowed

Reply:
https://github.com/kurator-dev/kurator/blob/c66a4fd4434fbbf3738eb21d005d10203e9de801/pkg/apis/cluster/v1alpha1/cluster_types.go#L222

will bock unsupported `Kind` in Webhook later.